ads/onead.md

```diff
@@ -0,0 +1,22 @@
+# OneAD
```

Comment:
Missing copyright on the top of the file

see: https://raw.githubusercontent.com/shopping24/amphtml/67cb7ea4b7ac75eabeb4605e1c41b9a7f0c8daef/ads/_ping_.md

Reply:
@torch2424 
1. added copyright
2. Sorry for the empty content, I've updated the material.
3. I notice that the device you use is not mobile simulator.
Please use the mobile simulator, since we only show ad in mobile device.

Thanks a lot for your quick response :)